From 236e3e153c48542319efdf84e9e58eb62c3109b9 Mon Sep 17 00:00:00 2001
From: Jakub Adam <jakub.adam@ktknet.cz>
Date: Thu, 15 Mar 2018 17:41:43 +0100
Subject: [PATCH] appshare: Fix black screen with Remmina v1.2.0-rcgit.27

"colordepth" config option has changed its default value, but it turns
out any setting other than "0" doesn't work with Sipe.

Fixes "[WARN][com.freerdp.gdi] - Invalid SurfaceCommand 0x00000005".
---
 src/core/sipe-appshare-remmina.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/sipe-appshare-remmina.c b/src/core/sipe-appshare-remmina.c
index eeef121..b7c891c 100644
--- a/src/core/sipe-appshare-remmina.c
+++ b/src/core/sipe-appshare-remmina.c
@@ -81,6 +81,7 @@ remmina_launch(struct sipe_rdp_client *client, GSocketAddress *listen_address,
 				      "scale=1\n"
 				      "aspectscale=1\n"
 				      "viewmode=1\n"
+				      "colordepth=0\n"
 				      "disableautoreconnect=1\n",
 				      alias ? alias : stream->call->with,
 				      address_string,
--
2.10.5.GIT
